client/service-container-chain/src/service.rs

```diff
@@ -308,8 +300,8 @@ pub fn container_chain_import_queue(
 fn start_consensus_container(
     client: Arc<ContainerChainClient>,
     backend: Arc<FullBackend>,
-    orchestrator_client: Arc<ParachainClient>,
-    orchestrator_tx_pool: Arc<FullPool<Block, ParachainClient>>,
+    // TODO: this should be generic, so we can use RelayChainInterface for solochain
+    collation_params: crate::spawner::CollationParams,
```

Comment:
What does this mean exactly? I see that we are already using the same `relay_chain_interface` object for both solochain and parachain modes, right? I didn't fully get the TODO meaning.

Reply:
That's comment is outdated, it was referring to the `orchestrator_client` field which I ended up removing, so I will remove the TODO as well.

Reply:
*fixed*